.github/workflows/integration.yml

```diff
@@ -187,3 +171,116 @@ jobs:
           -pull-request-number="${PULL_REQUEST_NUMBER}" \
           -init-result="${{ needs.init.result }}" \
           -plan-result="${{ needs.plan.result }}"
+
+  apply:
+    needs:
+      - 'init'
+      - 'plan'
+      - 'plan_success'
+    runs-on: 'ubuntu-latest'
+    strategy:
+      fail-fast: false
+      max-parallel: 100
+      matrix:
+        working_directory: ['./terraform/local-backend']
```

Comment:
This should be same as above taking dirs from init.

Reply:
done.